feed.11ty.js

```diff
@@ -23,8 +23,8 @@ exports.render = async function ({ permalink, collections }) {
     <?xml version="1.0" encoding="utf-8" ?>
     <feed xmlns="http://www.w3.org/2005/Atom" xml:base="${metadata.url}">
       <title>${metadata.title}</title>
-      <link href="${permalink | rss.absoluteUrl(metadata.url)}" rel="self"/>
-      <link href="${metadata.url}"/>
+      <link href="${permalink || rss.absoluteUrl(metadata.url)}" rel="self"/>
```

Comment:
Oops! Looks like I missed this one when converting from Nunjucks. It should actually be:

```js
rss.absoluteUrl(permalink, metadata.url)
```

Reply:
We're providing the `xml:base` so having a relative path in the `href` appears to be fully valid. But that does explain it, I was a little confused.